_download-meta/openttd-nightlies.md

```diff
@@ -1,15 +1,14 @@
 ---
-folder_old_infrastructure: nightlies/trunk
 ---
 
 OpenTTD requires a separate base graphics set to run, which can be downloaded through the installer (Windows) or on first run (Windows, Linux).
 In case you want to manually download and install it, some options are:
-* [OpenGFX (graphics)](../opengfx-releases/latest.html)
+* [OpenGFX (graphics)](https://www.openttd.org/downloads/opengfx-releases/latest.html)
 * The Transport Tycoon Deluxe data files can be used as base graphics set. You must own a version of Transport Tycoon Deluxe; they cannot be downloaded.
 
 Sound and music sets are recommended (but not necessary).
-[OpenSFX (sound)](../opensfx-releases/latest.html), and [OpenMSX (music)](../openmsx-releases/latest.html) can be installed via the installer or - if you own the CD - also the Transport Tycoon Deluxe sound and music.
+[OpenSFX (sound)](https://www.openttd.org/downloads/opensfx-releases/latest.html), and [OpenMSX (music)](https://www.openttd.org/downloads/openmsx-releases/latest.html) can be installed via the installer or - if you own the CD - also the Transport Tycoon Deluxe sound and music.
 
 Additional graphics, sound, and music sets can be downloaded via the ingame content download manager.
 
-Please refer to the [readme](https://proxy.binaries.openttd.org/openttd-nightlies/@@version@@/README.md) for more information.
+Please refer to the [readme](https://cdn.openttd.org/@@folder@@/README.md) for more information.
```

Comment:
Mildly similarly for cdn - maybe want a config variable instead?

Reply:
There is no staging variant of the CDN, so if you meant that, no, there is no need to do so :)